prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -248,6 +248,10 @@ ssize_t rxr_pkt_post_ctrl_once(struct rxr_ep *rxr_ep, int entry_type, void *x_en
 	ssize_t err;
 	fi_addr_t addr;
 
+	/* inject path should not have additional flags for send/recv operations */
```

Comment:
Since only acceptable flag is FI_MORE, maybe add an assertion here like `assert(flags == FI_MORE || flags == 0)`

Reply:
Why is FI_MORE the only acceptable flag?

Reply:
Currently the callers of this function set `flags` either 0 or `FI_MORE`. Within this function's level I do not see why `FI_MORE` is the only acceptable flag beyond 0. I think making sure flags is 0 in inject path may be enough.